worker/provisioner/container_initialisation_test.go

```diff
@@ -368,13 +368,17 @@ func (t toolsFinderFunc) FindTools(v version.Number, series string, arch string)
 }
 
 func getContainerInstance() (cont []ContainerInstance, err error) {
+	pkgs := [][]string{
+		{"qemu-kvm"},
+		{"qemu-utils"},
+		{"genisoimage"},
+		{"libvirt-bin"},
+	}
+	if runtime.GOARCH == arch.ARM64 {
```

Comment:
Made it depend on arch.HostArch() rather than runtime as the rest of the test does.